python/cudf/cudf/core/frame.py

```diff
@@ -1151,6 +1151,29 @@ def fillna(
             inplace=inplace,
         )
 
+    def bfill(self, value=None, method=None, axis=None, inplace=None, limit=None):
```

Comment:
(this feedback applies to `ffill()` as well)

The function `bfill()` is defined in Pandas for both the `Series` and  `DataFrame` classes. Right now, you have a `bfill()` method in both the `Series` and `Frame` classes, and they look identical.

Instead, you could have just one version of it under the `IndexedFrame` class. It will then be inherited by both `Series` and `DataFrame`.

The document [library_design.md](https://github.com/rapidsai/cudf/blob/branch-22.08/docs/cudf/source/developer_guide/library_design.md) goes into the details about each of these classes and when to place methods in which.